app/main.py

```diff
@@ -1,3 +1,28 @@
 def create_dict(*args):
-    # write your code here
-    pass
+    diction = {}
+    value = 0
+    for argument in args:
+        if isinstance(argument, int) or isinstance(argument, float)\
+                or isinstance(argument, str) or isinstance(argument, bool)\
+                or argument is None or callable(argument):
```

Comment:
it's not necessary but "isinstance" can take tuple of possible types as second argument, you can make it shorter